Makefile

```diff
@@ -61,7 +69,7 @@ export: VERSION.full man/charliecloud.1
 	tar Af main.tar bats.tar
 	tar --xform=s,^,charliecloud-$$(cat VERSION.full)/, \
             -rf main.tar \
-            man/*.1 VERSION.full
+            $^
```

Comment:
`$^` doesn't capture all the man pages since only `charliecloud.1` is depended upon.